bc/subscription/tasks.py

```diff
@@ -8,15 +13,34 @@ def process_filing_webhook_event(fwe_pk) -> FilingWebhookEvent:
     :return: A FilingWebhookEvent object that was updated.
     """
     filing_webhook_event = FilingWebhookEvent.objects.get(pk=fwe_pk)
-    try:
-        if filing_webhook_event.docket_id:
+    if filing_webhook_event.docket_id:
```

Comment:
Rather than doing a 30-line indent, you should do an early exit from the function here. Flip the logic, so you have:

```suggestion
    if not filing_webhook_event.docket_id:
        return filing_webhook_event
    
    # Then the rest can be outdented here
```